content/non-web-ict.md

```diff
@@ -90,6 +92,6 @@ WCAG2ICT is developed by the [WCAG2ICT Task Force](https://www.w3.org/WAI/GL/WCA
 
 ### Comments
 
-If you have any comments on the Additional Guidance sections of WCAG2ICT, please send them to the public mailing list <public-wcag2ict-comments@w3.org>.
+If you have any comments on the Additional Guidance sections of WCAG2ICT, @@ UPDATE with GitHub info @@ please send them to the public mailing list <public-wcag2ict-comments@w3.org>.
```

Comment:
```suggestion
If you have any comments on the Additional Guidance sections of WCAG2ICT, you can open an issue at https://github.com/w3c/wcag2ict/issues/new. It is free to create a GitHub account. If GitHub is not feasible, please send comments to the public mailing list <public-wcag2ict-comments@w3.org>.
```

Reply:
I used the more specific and updated wording from the e-mail announcement. :)